### PR TITLE
PlainMarkdownEditor highlighting

### PR DESCRIPTION
**Pull request by abaliunov-sc**
Fixing bug with highlighting

### CODE REVIEW

**Comment by abaliunov-sc** [comment]
See [interactive example](https://opuscapita.github.io/react-markdown/branches/markdown-it_tokeniser(slash)(hash)21/?currentComponentId=%40opuscapita%2Freact-markdown%2F1.1.4%2FMarkdownInput)

**Comment by nstoliar-sc** [comment]
"Error: `Mark.fromJS` requires a `type` string." appears while performing follow steps:
1. Add link in markdown using appropriate button on items pane;
2. try to input  value {\*\*test\*\*} in text part of link.
Result: error appeared.

File: package.json [hunk 1]
```diff
@@ -67,6 +69,8 @@
     "lodash-webpack-plugin": "0.11.4",
     "mocha": "3.4.2",
     "mocha-junit-reporter": "1.13.0",
+    "nodemon": "1.11.0",
```
Comment (kvolkovich-sc) [inline package.json]: Is `nodemon` used somewhere?

File: src/client/components/PlainMarkdownInput/PlainMarkdownInput.spec.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+    let nodes = editorState.document.nodes.asMutable();
+    let nodesSize = nodes.size;
+    expect(nodesSize).to.equal(1);
+
+    const currNode = nodes.get(0).asMutable();
+    expect(currNode.data.text).to.equal(nodeText);
+    expect(currNode.data.tokens).to.deep.equal(parse(nodeText));
+
+    // let wrapperInstance = wrapper.instance();
```
Comment (kvolkovich-sc) [inline src/client/components/PlainMarkdownInput/PlainMarkdownInput.spec.js]: Remove comment line

**Comment by abaliunov-sc** [comment]
Done

**Pull request merged**